### PR TITLE
Edits for 1.1 History, 1.2 for Azure ecosystem, and new 1.4 section for interfaces

### PR DESCRIPTION
**Pull request by rgward**
This PR includes:

Small edits for history
New section in 1.2 for Azure ecosystem
New 1.4 section for interfaces

### CODE REVIEW

File: azuresqlworkshop/01-IntroToAzureSQL.md [hunk 1]
```diff
@@ -115,10 +159,10 @@ In this section, you've learned about Azure SQL and the deployment options that
 
 
 If you want to dive deeper into the deployment options and how to choose, check out the following resources:  
-* [Blog announcement for Azure SQL](https://techcommunity.microsoft.com/t5/Azure-SQL-Database/Unified-Azure-SQL-experience/ba-p/815368) which explains and walks through Azure SQL and some of the resulting views and experiences available in the Azure portal.
```
Comment (amthomas46) [inline azuresqlworkshop/01-IntroToAzureSQL.md:118]: For some reason the `.` in this URL was removed? can you add it back and commit?

File: azuresqlworkshop/01-IntroToAzureSQL.md [hunk 2]
```diff
@@ -115,10 +159,10 @@ In this section, you've learned about Azure SQL and the deployment options that
...
-* [Choosing your database migration path to Azure](https://azure.microsoft.com/mediahandler/files/resourcefiles/choosing-your-database-migration-path-to-azure/Choosing_your_database_migration_path_to_Azure.pdf) is a white paper that talks about tools for discovering, assessing, planning and migrating SQL databases to Azure. This workshop will refer to it several times, and it's a highly recommended read. Chapter 5 deeply discusses choosing the right deployment option.  
+* [Choosing your database migration path to Azure](https://azure.microsoft.com/mediahandler/files/resourcefiles/choosing-your-database-migration-path-to-azure/Choosing_your_database_migration_path_to_Azure.pdf) is a white paper that talks about tools for discovering, assessing, planning and. migrating SQL databases to Azure. This workshop will refer to it several times, and it's a highly recommended read. Chapter 5 deeply discusses choosing the right deployment option.
```
Comment (amthomas46) [inline azuresqlworkshop/01-IntroToAzureSQL.md]: For some reason a `.` was moved `planning and. migrating`, can you remove it and commit?

File: azuresqlworkshop/01-IntroToAzureSQL.md (hunk 1 above)
Comment (rgward) [reply azuresqlworkshop/01-IntroToAzureSQL.md:118]: fixed

File: azuresqlworkshop/01-IntroToAzureSQL.md (hunk 2 above)
Comment (rgward) [reply azuresqlworkshop/01-IntroToAzureSQL.md]: fixed

**Pull request merged**